crates/astria-conductor/src/sequencer/mod.rs

```diff
@@ -160,8 +158,11 @@ impl Reader {
                     } else {
                         info!("sync finished successfully");
                     }
-                    let sync_done = sync_done.take().expect("channel should only be used once and only in this branch; this is a bug");
-                    let _ = sync_done.send(());
+                    // First sync at startup: notify conductor that sync is done.
+                    // Every resync after: don't.
```

Comment:
why do we only need to notify the first time?

Reply:
Because we don't have a mechanism to stop DA execution if the sequencer read goes down. Nor do I think we should (dunno what this should look like).

More technically speaking: we use a one shot channel to signal to conductor that it may spin up the DA reader after the first sequencer sync is done. It's an error to use the one shot channel again, so we have to protect against it.